src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5089,36 +5101,126 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_RefReturnOnlyParameter2" xml:space="preserve">
     <value>Cannot return by reference a member of parameter '{0}' through a ref parameter; it can only be returned in a return statement</value>
   </data>
+  <data name="WRN_RefReturnOnlyParameter" xml:space="preserve">
+    <value>This returns a parameter by reference '{0}' through a ref parameter; but it can only be returned in a return statement</value>
```

Comment:
It looks like the goal of these diagnostic messages was to remove terms like *cannot*, but we still have the phrase *can only be returned in a return statement* here. Maybe *can only be safely returned in a return statement*?